index.js

```diff
@@ -31,9 +31,15 @@ function fastifySensible (fastify, opts, next) {
 
   // TODO: benchmark if this closure causes some performance drop
   Object.keys(httpErrors).forEach(httpError => {
-    fastify.decorateReply(httpError, function (message) {
-      this.send(httpErrors[httpError](message))
-    })
+    if (httpError === 'notFound') {
+      fastify.decorateReply(httpError, function (message) {
+        this.callNotFound()
```

Comment:
I don't think this is needed. The `callNotFound` invokes the routes 404 handlers, while here a developer can return a 404 for many other reasons.

Reply:
Without the change https://github.com/fastify/fastify-sensible/blob/master/test/httpErrorsReply.test.js#L78 fails

Is there any other way?

Reply:
It's correct, in Fastify v2 we are no longer overriding the not found message (https://github.com/fastify/fastify/pull/1084).
You just need to update the test and remove the `if (code === '404')` check :)